vip_spec.xsd

```diff
@@ -298,6 +298,9 @@ reference elements for polling location
         <xs:element name="Party">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: 1622.2 has abbreviation as a string, not InternationalizedText.
+                   Which makes more sense?
```

Comment:
In my mind, it all depends on whether non-English ballots display abbreviations in languages other than English.  I don't have access to current sample ballots, though, so I can't provide any data points.